activitysim/examples/prototype_mtc_extended/configs/disaggregate_accessibility.yaml

```diff
@@ -0,0 +1,159 @@
+# Sampling size. 0 = no limit
+# can be whole integer value or a sample rate (percent of zones)
+# zero or missing defaults to full sample!
+DESTINATION_SAMPLE_SIZE: 0.5
+ORIGIN_SAMPLE_SIZE: 5000
+
+# select origin zones weighted by population (or another landuse variable)
+ORIGIN_WEIGHTING_COLUMN: TOTPOP
+
+# Specify the tables to be created and  their variables.
+# Single values are constants
+# Lists are varying and will generate the cartesian product (i.e., all possible non-repeating combinations)
+
+# Example result for households below:
+#hhid   veh hinccat1    hinc    hworkers    persons hht  bldgsz
+# 1     0   1           14000   1           2       1    2
+# 2     1   1           14000   1           2       1    2
+# 3     2   1           14000   1           2       1    2
+# 4     0   2           67000   1           2       1    2
+# 5     1   2           67000   1           2       1    2
+# 6     2   2           67000   1           2       1    2
+# ....
+
+#Note: parameters in ALL CAPS below are required by the program for synthesis
+CREATE_TABLES:
+    PROTO_HOUSEHOLDS:
+        index_col: proto_household_id
+        zone_col: home_zone_id
+        rename_columns:
+            zone_id: home_zone_id
+        VARIABLES:
+            hinccat1: [1, 2, 3, 4]          # Income categories
+            hworkers: 1                  # Household workers
+            veh: [0, 1, 2]               # Household vehicles
+            persons: 2                   # Two persons household
```

Comment:
You're correct, this was created as specified in the 'Disaggregate Accessibilities Implementation Plan' (https://github.com/ActivitySim/activitysim/wiki/Project-Meeting-2022.03.22). 

You can change the household size, but you can't have multiple household sizes. E.g., you can't have a 2 person household and a 3 person household because persons must then be mapped to the households and this greatly complicates the generation procedure.

I agree that users may want this functionality, but is a bit out of scope. However, I think a simple solution for complicated cases would be to just let users optionally create their own person/household/tour templates tables in a csv that gets read in. Will work on adding this.